sample-compose/detekt-config.yml

```
@@ -189,7 +189,7 @@ naming:
   FunctionNaming:
     active: true
     functionPattern: '^([a-z$][a-zA-Z$0-9]*)|(`.*`)$'
```

The migration doc says we should use this rule instead.

```suggestion
    functionPattern: '[a-zA-Z][a-zA-Z0-9]*'
```

---

Yeah, I agree @luongvo  but If we update this, I think we should update for `template-xml` and `sample-xml` as well 🚀

---

@Wadeewee I'm not sure. I think we should focus on updating Detekt rules for Compose in this PR only 🤔